.flake8

```diff
@@ -4,3 +4,4 @@ exclude =
     __pycache__,
     build
 max-complexity = 10
+max-line-length = 120
```

Comment:
We should open an issue and a separate PR for this change. If we decide to do this, we have to update the black and isort configs in the pyproject.toml file.